template/src/App.js

```diff
@@ -1,23 +1,46 @@
 import React from 'react';
 import logo from './logo.svg';
+import { Counter } from './features/counter';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
+        <Counter />
         <p>
           Edit <code>src/App.js</code> and save to reload.
         </p>
-        <a
```

Comment:
Can we leave the "Learn React" link in here? That's over half of what the final app will be anyway :)